views.go

```diff
@@ -580,6 +567,64 @@ func (lk *logkeeper) viewBucketLogs(r *http.Request, buildID string, testID stri
 	}, nil
 }
 
+func writeRawLines(w http.ResponseWriter, resp *logFetchResponse) error {
+	var (
+		numLines    int
+		totalSize   int
+		maxLineSize int
+		minLineSize = maxLogBytes + len("\n")
+	)
+
+	var hasLines bool
+	for line := range resp.logLines {
+		hasLines = true
```

Comment:
do we want to write `lk.render.WriteJSON(w, http.StatusOK, nil)` for the `!hasLines` case?
Or at least to send a 200 response code?

Reply:
It automatically returns a 200 and empty response, I checked in staging

Reply:
Ah nice!
I think [I found it](https://github.com/golang/go/blob/ba76567bc2500204432ed8a5cb28848410e74447/src/net/http/server.go#L1624-L1626).

Reply:
Good find!